packages/liveblocks-client/e2e/list.test.ts

```diff
@@ -26,16 +21,6 @@ describe("LiveList conflicts", () => {
           await wsUtils.flushSocket2Messages();
 
           assert({ list: ["A", "B"] });
-
```

Comment:
I removed these assertions to simplify the tests.

Updates are not strictly validated. They are applied on an immutable state, then we assert that this immutable state is consistent with the LiveStructure.